docs-conceptual/azurermps-5.7.0/install-azurerm-ps.md

```diff
@@ -38,161 +40,101 @@ PowerShellGet 1.6.0   C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1
 PowerShellGet 1.0.0.1 C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PowerShellGet.psd1
 ```
 
-You need PowerShellGet version 1.1.2.0 or higher. To update PowerShellGet, use the following command:
+If you need to update your installation of PowerShellGet, run the following command:
 
 ```powershell
 Install-Module PowerShellGet -Force
 ```
 
-If you do not have PowerShellGet installed, see the [How to get PowerShellGet](#how-to-get-powershellget)
-section of this article.
+If you don't have PowerShellGet installed, follow the instructions in the table below for your system.
+
+|Scenario|Install instructions|
+|---|---|
+|Windows 10<br/>Windows Server 2016|Built into Windows Management Framework (WMF) 5.0 included in the OS|
+|Upgrade to PowerShell 5|<ol><li>[Install the latest version of WMF](https://www.microsoft.com/en-us/download/details.aspx?id=54616)</li><li>Run the following command:<br/>```Install-Module PowerShellGet -Force```</li></ol>|
```

Comment:
You don't need triple-backticks for inline code blocks. I am not sure why they were written that way to begin with. You can leave it. It will work either way.